### DISCUSSION TITLE
Midi dummy

### DISCUSSION THREAD

**Discussion by Wapata** [shanteacontrols/OpenDeck#146] [Ideas]
Hi,
One is not able to see the potential of OpenDeck if it doesn't have a board with OpenDeck Firmware. Do you think you can add a switch to the Configurator interface just to skip the "No OpenDeck board found. Please connect the board in order to use the interface." message and see ?

I had the chance to find a board in my garage, but even now, just to navigate thru OpenDeck Configurator for asking a question... I should have my OpenDeck device next to me.

For new futur-user it should be cool. And yes, i did see your youtube video of you navigate into the menus but it wasn't the same as discover it by myself.

**Comment by paradajz** [comment]
That is not possible. What's displayed in the UI depends entirely on what the board reports, so you do need to have board to use the UI.

**Comment by Wapata** [comment]
Thank you for the answer. Have a great day ;-)
